libs/partners/robocorp/langchain_robocorp/toolkits.py

```diff
@@ -227,6 +230,8 @@ def _action_request(self, endpoint: str, **data: dict[str, Any]) -> str:
             "Authorization": f"Bearer {self.api_key}",
             "Content-Type": "application/json",
         }
+        if self.context is not None:
```

Comment:
```suggestion
        if self.context:
```

Reply:
I was thinking what if someone wants to explicitly pass en empty string as context (not sure why they would but just to keep that possibility).